configure.ac

```diff
@@ -736,6 +736,15 @@ if test -z "$timer_type" ; then
 fi
 AC_MSG_NOTICE([Timer type selected is $timer_type])
 
+# check __builtin_expect
+AC_TRY_COMPILE(
```

Comment:
FYI, `AC_TRY_COMPILE` is "obsolete" according to the autoconf manual. `AC_COMPILE_IFELSE` should be used instead. https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Obsolete-Macros.html

Reply:
Thank you! The current `configure.ac` is based on `AC_TRY_COMPILE`. However, it is obsolete even in 2.6.3 (`AC_PREREQ(2.63)` is specified in `configure.ac`). I will check obsolete configure macros and update them in a separate PR if needed.

Reply:
👍